backend/api/Pims.Api.csproj

```diff
@@ -8,6 +8,7 @@
     <AssemblyVersion>0.1.0.0</AssemblyVersion>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <ProjectGuid>FA2CFDF0-DB90-4D54-813D-8BAD039A74F6</ProjectGuid>
```

Comment:
SonarQube for NET Core cannot scan projects unless they have valid GUIDs. NET Core doesn't use them but we need to add it to please SonarQube...

Reply:
Just a warning about these.  If for some reason our branches get out of sync (different devs using different guids), it causes havoc when using Visual Studio.